site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -104,10 +105,39 @@ spec:
     - matchLabels:
         agones.dev/fleet: v2
 ```
+{{% /feature %}}
 
+{{% feature publishVersion="1.17.0" %}}
+```yaml
+apiVersion: "allocation.agones.dev/v1"
+kind: GameServerAllocation
+spec:
+# Desprecated, use field selectors instead.
```

Comment:
This example won't actually work, since it shows setting both the old and new fields. I'm inclined to have the website docs only show the new fields starting with the next release. Note that this is different that my suggestion on the reference page, which should show _all_ of the fields for completeness. 

@markmandel WDYT?

Thinking about this more, if we do start only documenting the new field, then we should add a warning here that this example only works on agones 1.17+ with a link to the website docs for the 1.16 release with the old example.

Reply:
The example here kind of works. If both new and old fields are set, starting from 1.17, agones will only take the new field into consider. That's said, this example is equivalent to remove the `required` and preferred` fields.